examples/nextjs-with-typescript/pages/MuxPlayer.tsx

```diff
@@ -131,160 +143,90 @@ const PlayerSizeHeights = {
 };
 
 const DEFAULT_INITIAL_STYLES_STATE = {
-  // width: PlayerSizeWidths[MediaChromeSizes.XS] + 'px',
-};
-
-const DEFAULT_INITIAL_OPTION_STYLES_STATE = {
-  '--player-height': PlayerSizeHeights[MediaChromeSizes.LG] + 'px'
-  // '--player-height': PlayerSizeHeights[MediaChromeSizes.XS] + 'px'
+  width: `${PlayerSizeWidths[MediaChromeSizes.LG] + 1} + px`,
+  // width: PlayerSizeWidths[MediaChromeSizes.LG] + 'px',
+  // height: PlayerSizeHeights[MediaChromeSizes.LG] + 'px',
 };
 
-const reducer = (state: Partial<{ [k: string]: any }>, action): Partial<{ [k: string]: any }> => {
-  const { type, value } = action;
-  switch (type) {
-    case ActionTypes.UPDATE: {
-      return {
-        ...state,
-        ...value,
-      };
-    }
-    default: {
-      return state;
-    }
-  }
-};
-
-const toInitialState = (selectedAsset: typeof mediaAssetsJSON[0] | undefined, mediaAssets: typeof mediaAssetsJSON, query: NextParsedUrlQuery) => {
+const toInitialState = (
```

Comment:
This should probably be improved to work more directly with the abstracted state mgmt stuff in `page-state/index.ts` but I didn't want to do any unnecessary architecture overhauls in this round of migration.